addon/helpers/split-grid-detail-content.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+/**
+ * @module helpers
+ */
+
+/**
+ * Lookup and render the template for a split-grid row's record
+ *
+ * @function split-grid-detail-content
+ * @param    {object} options - The Ember-supplied bound options representing the calling view's state
```

Comment:
All of the other files in this repository use Ember.Object. Please change the occurrence for this property to be consistent with the rest of the codebase. If it is desired to move away from the "Ember." syntax create an issue to be worked on separately.